python/python/glide/async_commands/core.py

```diff
@@ -1368,3 +1373,111 @@ async def invoke_script(
                 ["foo", "bar"]
         """
         return await self._execute_script(script.get_hash(), keys, args)
+
+    @dataclass
+    class PubSubMsg:
```

Comment:
General note: lets move all pubsub code to a dedicated pubusb.py file under the async_commands folder